Jira.md

```diff
@@ -0,0 +1,29 @@
+# MetaBrainz bug tracking
+
+MetaBrainz uses Jira to track bugs. It is hosted at https://tickets.metabrainz.org/
+
+Each main MetaBrainz project has its own project in Jira.
+
+### Working on tickets
+
+If you want to work on a ticket, you don't need to ask for permission. Just check
+that the ticket hasn't been assigned to any other user, and assign it to yourself
+by clicking on the _Assign_ or _Assign to me_ link.
+
+If you want to work on a ticket which has been assigned to someone else, and it doesn't
+look like any work has been done for a while, you can post a question asking them if
+they are still going to complete the work.
+
+If you have any questions about a ticket, add a comment. Project leads will automatically
+get sent an email informing them of your comment, so you don't need to follow it up
```

Comment:
My reason for this is that I want to stop people from telling me on IRC "hey I did something, can you look". If this isn't global we should do something else. Perhaps just saying this is good enough?